docs/testing/installation-manager.md

```diff
@@ -169,6 +169,21 @@ most use-cases but there are some limitations. For example, if you're creating
 database tables or performing complex SQL queries, you may run into issues and
 are better off not using SQLite.
```

Comment:
Can one install a plugin using the manager? It is currently very hacky in a CI context to install dependencies. See [here](https://github.com/alleyinteractive/elasticsearch-extensions/blob/main/.github/workflows/unit-tests.yml#L78-L84).

Since the Mantle test directory is not in the same folder as the project in a CI.

I also wonder what's the difference between installing with `with_object_cache` and `with_vip_mu_plugins`, and adding via https://github.com/alleyinteractive/mantle-ci I'm guessing we don't need to add it via the mantle CI if adding it via the manager?!

Reply:
My goal is to not require anybody to need to run the `mantle-ci` script themselves and only need to run `composer phpunit`.